### DISCUSSION TITLE
antd-pro  loading页怎么去掉或者改为自己的文字

### DISCUSSION THREAD

**Discussion by qyuja** [ant-design/ant-design-pro#7949] [Q&A]
![image](https://user-images.githubusercontent.com/63438413/104443033-d6c93080-55d0-11eb-9b23-97a85509759d.png)
感觉这个页面应该是在securityLayout之前的页面，没有找到 有哪位大佬知道吗

**Comment by drizzlesconsin** [comment]
Pro 文本在这里 https://github.com/ant-design/ant-design-pro/blob/master/public/pro_icon.svg

可以在这里修改 https://github.com/ant-design/ant-design-pro/blob/master/src/pages/document.ejs#L193
